gradle/libs.versions.toml

```diff
@@ -14,7 +14,7 @@ snakeyaml         = "2.1"
 icu4j             = "77.1"
 supercsv          = "2.4.0"
 log4j             = "2.21.0"
-slf4j             = "1.7.36"
+slf4j             = "2.0.17"
```

Comment:
@reta @willyborankin I know this is a major version bump, would there be any considerations for downstream plugins consuming this?